### PR TITLE
feat: 3id, did s3 resolve, verify

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,13 @@
   },
   "homepage": "https://github.com/3box/3box-verifications#readme",
   "dependencies": {
+    "3id-resolver": "0.0.6",
     "analytics-node": "^3.4.0-beta.1",
     "aws-sdk": "^2.442.0",
     "did-jwt": "^0.1.3",
+    "ipfs-s3-dag-get": "^0.1.12",
     "js-sha256": "^0.9.0",
+    "muport-did-resolver": "^0.3.1",
```
Comment (oed) [inline package.json]: 😕 I can't believe we missed this before

File: src/lib/claimMgr.js [hunk 2]
```diff
@@ -7,12 +16,16 @@ class ClaimMgr {
   }
 
   isSecretsSet () {
-    return (this.signerPrivate !== null || this.signerPublic !== null)
+    return ( this.ipfs !== null && (this.signerPrivate !== null || this.signerPublic !== null))
```
Comment (oed) [inline src/lib/claimMgr.js]: Should this not be `and` and not `or` for all? I.e. if any one of these variables are not set then this should return false?

**Comment by zachferland** [comment]
there is no jwt/decode for the twitter handler, mean something else?

File: src/lib/claimMgr.js (hunk 2 above)
Comment (zachferland) [reply src/lib/claimMgr.js]: 👍

**Comment by zachferland** [comment]
reviewed again, made fixes, and tested, all set to merge/release cc @oed

**Pull request merged**